workflow/rocoto/gefs_tasks.py

```diff
@@ -3,7 +3,6 @@
 import rocoto.rocoto as rocoto
 from datetime import datetime, timedelta
 
-
```

Comment:
This needs to be restored.  `pynorms` is failing.

Reply:
copy over from develop and restore it.